### PR TITLE
Rebase the repo

### PR DESCRIPTION
**Pull request by AdfPizarro**
Add css styling, created a new branch to follow the github flow
and closed the previous pull request in the wrong branch

https://github.com/AdfPizarro/awesome_books/pull/6

### CODE REVIEW

File: dom.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ class Dom { // eslint-disable-line no-unused-vars
     this.getBookList = document.getElementById('bookList');
   }
 
+
```
Comment (rloterh) [inline dom.js:6]: - [ ] ESlint check is failing on this line. For cleaner code kindly ensure not leave empty lines which are not serving any purpose. Also kindly endeavor to fix **all** detected ESlint errors.

**ESlint Errors:**
image

File: index.html [hunk 2]
```diff
@@ -3,6 +3,13 @@
...
     <title>Awesome books</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link rel="preconnect" href="https://fonts.googleapis.com">
+
+
+<link href="https://fonts.googleapis.com/css2?family=MonteCarlo&family=Noto+Sans+TC:wght@500&family=Roboto:wght@300&display=swap" rel="stylesheet">
   </head>
   <body>
```
Comment (rloterh) [inline index.html:14]: - [ ]  Your app interface does not match with the given [wireframe template](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m3_style_core_elements.md). We actually want your design to as much as possible follow the provided wireframe template.. Kindly ensure to improve upon this aspect so as to achieve a cleaner interface.

**Your design interface:**
image

**The original wireframe:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Mina & Adrian, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 


## Required Changes ♻️
- [ ] _Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Kindly check this comments marked [OPTIONAL]:

- [OPTIONAL]
Kindly provide **full name** of `Mina` and _Authors_ section of your ReadMe so as to make your ReadMe appear more professional.


Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abruzy — APPROVED** [review]
Hi Team 👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

_[OPTIONAL]_

- clear the input field after submission of the form, it would improve the user experience of your application

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**